### PR TITLE
[BEAM-2317] Fix GroupAlsoByWindowEvaluatorFactory doc

### PR DESCRIPTION
**Pull request by tgroh**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [x] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [x] Make sure tests pass via `mvn clean verify`.
 - [x] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [x] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---

### CODE REVIEW

**Comment by tgroh** [comment]
R: @manuzhang

**Comment by tgroh** [comment]
retest this please

File: runners/direct-java/src/main/java/org/apache/beam/runners/direct/GroupAlsoByWindowEvaluatorFactory.java [hunk 1]
```diff
@@ -92,8 +90,8 @@ public void cleanup() {}
   }
 
   /**
-   * A transform evaluator for the pseudo-primitive {@link GroupAlsoByWindow}. Windowing is ignored;
-   * all input should be in the global window since all output will be as well.
+   * A transform evaluator for the pseudo-primitive {@link DirectGroupAlsoByWindow}. Windowing is
+   * ignored; all input should be in the global window since all output will be as well.
```
Comment (manuzhang) [inline runners/direct-java/src/main/java/org/apache/beam/runners/direct/GroupAlsoByWindowEvaluatorFactory.java]: Windowing is ignored in `DirectGroupAlsoByWindow` ?

Comment (tgroh) [inline runners/direct-java/src/main/java/org/apache/beam/runners/direct/GroupAlsoByWindowEvaluatorFactory.java]: Fixed.

**Comment by manuzhang** [comment]
LGTM

**Pull request closed without merging**